.github/commands.json

```diff
@@ -169,6 +169,19 @@
 		"reason": "completed",
 		"addLabel": "unreleased"
 	},
+	{
+		"type": "comment",
+		"name": "spam",
+		"allowUsers": [
+			"cleidigh",
+			"usernamehw",
+			"gjsjohnmurray",
+			"IllusionMH"
+		],
+		"action": "close",
+		"reason": "not_planned",
+		"addLabel": "spam"
```

Comment:
@gjsjohnmurray could you please change the label to "invalid"?

Reply:
Done